section1/exercises/variables.js

```diff
@@ -17,39 +17,59 @@ console.log(name);
 
 // EXAMPLE: Write code below to add 2 to the variable `students` and
 // log the result:
-var enrolledStudents = 22 + 2;
-console.log(enrolledStudents);
+var enrolledStudents = 22;
+console.log(enrolledStudents + 2);
 
 // YOU DO:
 // Write code below to save the string 'Harry Potter must not return to Hogwarts!'
 // log that variable to the console.
-
+var message = 'Harry Potter must not return to Hogwarts!';
+console.log(message);
 
 // YOU DO:
-// Declare three variables, named `firstName`, `isHungry` and `numberOfPets`. 
+// Declare three variables, named `firstName`, `isHungry` and `numberOfPets`.
   // Store the appropriate data types in each.
 // log all three variables to the console.
+var firstName = 'Lidia';
+console.log(firstName);
+
+var isHungry = 'true';
```

Comment:
For this and line 58, you'll actually want to remove the quotes so that the `true` can be read by JavaScript as a boolean as opposed to reading it as a String!